test/libsolidity/util/TestFileParserTests.cpp

```diff
@@ -786,14 +786,6 @@ BOOST_AUTO_TEST_CASE(call_ether_type_invalid)
 	BOOST_REQUIRE_THROW(parse(source), langutil::Error);
 }
 
-BOOST_AUTO_TEST_CASE(call_hex_number_invalid)
-{
-	char const* source = R"(
-		// f(bytes32, bytes32): 0x616, 0x042 -> 1
-	)";
-	BOOST_REQUIRE_THROW(parse(source), langutil::Error);
-}
-
```

Comment:
If I understood correctly, can this be changed to a `BOOST_REQUIRE_SUCCESS`?

Reply:
Ah, nice one. I added another test based on the one removed which checks oddly-sized hex number literals.